src/main.c

```diff
@@ -1683,6 +1683,59 @@ static gboolean service_start(int argc, char **argv)
 	return TRUE;
 }
 
+static gboolean mount_start(int argc, char **argv)
+{
+	g_autofree gchar *bundlelocation = NULL;
+	g_autoptr(RaucBundle) bundle = NULL;
+	GError *error = NULL;
+	gboolean res = FALSE;
+
+	if (argc < 3) {
+		g_printerr("A file name must be provided\n");
+		goto out;
+	}
+
+	if (argc > 3) {
+		g_printerr("Excess argument: %s\n", argv[3]);
+		goto out;
+	}
+
+	bundlelocation = resolve_bundle_path(argv[2]);
+	if (bundlelocation == NULL)
```

Comment:
Is there some error that can be printed in this case?

Reply:
``resolve_bundle_path()`` prints an error. I've added a comment to the ``goto out;`` line.